spec/models/review_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
If you're going to use one of the more advanced generators make sure to delete any files you don't use.